### PR TITLE
Add UDS set up

### PR DESCRIPTION
**Pull request by aquiladayc**
### What does this PR do? What is the motivation?
Adding UDS set up example for Docker 

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/containers/docker/apm.md [hunk 1]
```diff
@@ -291,9 +291,40 @@ from ddtrace import tracer
...
+Refer to the [language-specific APM instrumentation docs][1] for tracer setting.
+
 ## Further Reading
 
 {{< partial name="whats-next/whats-next.html" >}}
 
+
+[1]: /tracing/setup/
 [1]: https://app.datadoghq.com/organization-settings/api-keys
 [2]: /tracing/guide/security/#replace-rules
```
Comment (drichards-87) [inline content/en/containers/docker/apm.md:330]:
```suggestion
[1]: https://app.datadoghq.com/organization-settings/api-keys
[2]: /tracing/guide/security/#replace-rules
[3]: /tracing/setup/
```

File: content/en/containers/docker/apm.md [hunk 2]
```diff
@@ -291,9 +291,40 @@ from ddtrace import tracer
...
+              gcr.io/datadoghq/agent:latest
+# Application
+docker run -d --name app \
+              --network <NETWORK_NAME> \
+              -v /var/run/datadog/:/var/run/datadog/ \
+              -e DD_TRACE_AGENT_URL=unix:///var/run/datadog/apm.socket \
+              company/app:latest
+```
+
+Refer to the [language-specific APM instrumentation docs][1] for tracer setting.
```
Comment (drichards-87) [inline content/en/containers/docker/apm.md:321]:
```suggestion
Refer to the [language-specific APM instrumentation docs][3] for tracer settings.
```

File: content/en/containers/docker/apm.md [hunk 3]
```diff
@@ -291,9 +291,40 @@ from ddtrace import tracer
 tracer.configure(hostname='172.17.0.1', port=8126)
 ```
 
+### Unix Domain Socket (UDS)
+To submit traces via socket, it should be mounted to the Agent container and your application container.
```
Comment (drichards-87) [inline content/en/containers/docker/apm.md:295]: Can you clarify what "it" is in the context of "being mounted to the Agent container and your application container"?

**Review by drichards-87 — CHANGES_REQUESTED** [review]
Left you some minor feedback.

File: content/en/containers/docker/apm.md (hunk 3 above)
Comment (aquiladayc) [reply content/en/containers/docker/apm.md:295]: "it" is socket. Otherwise it's not accessible by the agent and tracer.

**Comment by aquiladayc** [comment]
Thanks @drichards-87 I reflected it/responded it

File: content/en/containers/docker/apm.md (hunk 3 above)
Comment (drichards-87) [inline content/en/containers/docker/apm.md:295]:
```suggestion
To submit traces via socket, the socket should be mounted to the Agent container and your application container.
```

**Review by drichards-87 — APPROVED** [review]

**Comment by drichards-87** [comment]
Thanks for clarifying @aquiladayc. I made a small update to your PR, but it looks good otherwise. Let me know if this is ready to be merged.

**Comment by aquiladayc** [comment]
Thank you @drichards-87, looks all good for me. Please merge at your convenience.

**Pull request merged**